lib/macho/load_commands.rb

```diff
@@ -399,6 +339,18 @@ def initialize(view, cmd, cmdsize, name, timestamp, current_version, compatibili
       @current_version = current_version
       @compatibility_version = compatibility_version
     end
+
+    # @param endianness [Symbol, nil] the endianness of the serialized string,
+    #  or nil to use the view's endianness
+    # @return [String] the serialized fields of the load command
+    # @api private
+    def serialize(endianness = nil)
+      endianness ||= view.endianness
```

Comment:
> To me, the easiest solution seems to be to just pass the MachOFile instance

I agree that this would be the easiest solution, but I think that it establishes a confusing object relationship - we'd be passing in a whole file instance to create an independent load command whose only relationship with the file is a similarity of intrinsics (endianness, rounding size, etc). 

It would also require us to do something like `lc.serialize(this)` when dealing with created load commands inside a `MachOFile`, and I think that's pretty inelegant compared to `lc.serialize(endianness)`.